### ISSUE TITLE
Packages that do not upgrade o report incorrect package version 

### ISSUE THREAD

**Issue by musm** [microsoft/winget-pkgs#18417]
The following packages
```
Cisco Webex Meetings              Cisco.CiscoWebexMeetings 41.5.4       41.5.6.9    winget
Geekbench 5                       PrimateLabs.Geekbench.5  Unknown      5.4.1       winget
Mozilla Firefox                   Mozilla.Firefox          89.0         89.0.1      winget
WhatsApp                          WhatsApp.WhatsApp        2.2121.6     2.2123.7    winget
Microsoft Windows Desktop Runtime Microsoft.dotnetRuntime  3.1.16.30112 5.0.7.30113 winget
```

Prepetually state that they are upgradable, despite upgrading them multiple times. I suspect there is an issue somewhere with the packages reporitng their version numbers.

**Comment by OfficialEsco** [comment]
TLDR; This is a winget CLI issue, the package versions are correct

```
PS C:\Users\Esco> winget install Cisco.CiscoWebexMeetings; winget install PrimateLabs.Geekbench.5; winget install Mozilla.Firefox ; winget install WhatsApp.WhatsApp ; winget install Microsoft.dotnetRuntime
Found Cisco Webex Meetings [Cisco.CiscoWebexMeetings]
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://akamaicdn.webex.com/client/WBXclient-41.5.6-9/webexapp.msi
  ██████████████████████████████   130 MB /  130 MB
Successfully verified installer hash
Starting package install...
Successfully installed
Found Geekbench 5 [PrimateLabs.Geekbench.5]
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://cdn.geekbench.com/Geekbench-5.4.1-WindowsSetup.exe
  ██████████████████████████████   119 MB /  119 MB
Successfully verified installer hash
Starting package install...
Successfully installed
Found Mozilla Firefox [Mozilla.Firefox]
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://download-installer.cdn.mozilla.net/pub/firefox/releases/89.0.1/win64/en-US/Firefox%20Setup%2089.0.1.msi
  ██████████████████████████████  53.3 MB / 53.3 MB
Successfully verified installer hash
Starting package install...
Successfully installed
Found WhatsApp [WhatsApp.WhatsApp]
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://web.whatsapp.com/desktop/windows/release/x64/WhatsAppSetup.exe
  ██████████████████████████████   124 MB /  124 MB
Successfully verified installer hash
Starting package install...
Successfully installed
Found Microsoft Windows Desktop Runtime [Microsoft.dotnetRuntime]
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://download.visualstudio.microsoft.com/download/pr/2b83d30e-5c86-4d37-a1a6-582e22ac07b2/c7b1b7e21761bbfb7b9951f5b258806e/windowsdesktop-runtime-5.0.7-win-x64.exe
  ██████████████████████████████  52.3 MB / 52.3 MB
Successfully verified installer hash
Starting package install...
Successfully installed
```

```
PS C:\Users\Esco> winget list
Name                                       Id                                         Version          Available Source
-----------------------------------------------------------------------------------------------------------------------
Geekbench 5                                PrimateLabs.Geekbench.5                    Unknown          5.4.1     winget
Mozilla Firefox                            Mozilla.Firefox                            89.0.1
WhatsApp                                   WhatsApp.WhatsApp                          2.2123.7
Microsoft Windows Desktop Runtime          Microsoft.dotnetRuntime                    5.0.7.30113
Cisco Webex Meetings                       Cisco.CiscoWebexMeetings                   41.5.6.9
```

Geekbench is not reporting its version so that is a Geekbench issue.
```
PS C:\Users\Esco> winget upgrade
Name        Id                      Version Available Source
------------------------------------------------------------
Geekbench 5 PrimateLabs.Geekbench.5 Unknown 5.4.1     winget
```

![image](https://user-images.githubusercontent.com/15158490/122797339-ab551b00-d2bf-11eb-931f-3bf526e6992e.png)
